airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/streams.py

```diff
@@ -482,6 +482,7 @@ class BasicReports(IncrementalTiktokStream, ABC):
 
     schema_name = "basic_reports"
     report_granularity = None
+    include_deleted = False
```

Comment:
```suggestion
```
No need to define a class wide variable if you're defining an instance wide variable

Reply:
done.